src/App.js

```diff
@@ -14,15 +14,15 @@ import mockCats from "./mockCats"
 
 const App = () => {
   const [cats, setCats] = useState(mockCats)
-  console.log(cats)
+  // console.log(mockCats)
 
   return (
 <>
 <Header />
 <Routes>
   <Route path="/" element={<Home />} />
-  <Route path="/catindex" element={<CatIndex />} />
-  <Route path="/catshow" element={<CatShow />} />
+  <Route path="/catindex" element={<CatIndex cats={mockCats} />} />
```

Comment:
Since mockCats is set to state, cats from state should be passed to the component.